tracer/xray.go

```
@@ -0,0 +1,69 @@
+
+type Segment struct {
+	segment *xray.Segment
+}
+
+type LambdaContext interface {
+	EventContext() context.Context
+	Bind(v any) error
+}
+
+func init() {
+	xray.Configure(xray.Config{
```

golangci-lint (1.64.8)

Error return value of `xray.Configure` is not checked

(errcheck)